.gitignore

```diff
@@ -1,7 +1,10 @@
 .idea/
 .vscode/
+*.pyc
 *.iml
 .env
 .DS_Store
 venv/
 .pytest_cache/
+__pycache__/
+*.py[cod]
```

Comment:
If you decided to add gitignore, do it properly. Add a new line at the end of file

Reply:
added a new line at the end